modern/src/App.js

```diff
@@ -141,7 +141,8 @@ function Layout({
     return null;
   }
 
-  const image = node.js_imageLookup(background);
+  const sharedMakiTree = useSelector(Selectors.getSharedMakiTree);
+  const image = node.js_imageLookup(background, sharedMakiTree);
```

Comment:
I've changed `js_imageLookup`, but the first thing it will do is search its own maki tree using its own node references, only if it fails to find the image will it search the `sharedMakiTree`